src/contract.rs

```diff
@@ -311,10 +321,14 @@ pub fn execute_create_stream(
         config.stream_creation_denom,
         config.stream_creation_fee,
         config.exit_fee_percent,
+        min_price,
     );
     let id = next_stream_id(deps.storage)?;
     STREAMS.save(deps.storage, id, &stream)?;
 
+    let threshold_state = ThresholdState::new();
+    threshold_state.set_treshold_if_any(stream.clone(), id, deps.storage)?;
```

Comment:
Grammar cop is here 👮‍♀️
```suggestion
    threshold_state.set_treshold_if_any(stream.clone(), id, deps.storage)?;
```